icu4j/main/tests/core/src/com/ibm/icu/dev/test/util/ULocaleTest.java

```diff
@@ -886,35 +883,6 @@ public void TestObsoleteNames(){
     @Test
     public void TestCanonicalization(){
         final String[][]testCases = new String[][]{
-                { "ca_ES_PREEURO", "ca_ES_PREEURO", "ca_ES@currency=ESP" },
-                { "de_AT_PREEURO", "de_AT_PREEURO", "de_AT@currency=ATS" },
-                { "de_DE_PREEURO", "de_DE_PREEURO", "de_DE@currency=DEM" },
-                { "de_LU_PREEURO", "de_LU_PREEURO", "de_LU@currency=EUR" },
-                { "el_GR_PREEURO", "el_GR_PREEURO", "el_GR@currency=GRD" },
-                { "en_BE_PREEURO", "en_BE_PREEURO", "en_BE@currency=BEF" },
-                { "en_IE_PREEURO", "en_IE_PREEURO", "en_IE@currency=IEP" },
-                { "es_ES_PREEURO", "es_ES_PREEURO", "es_ES@currency=ESP" },
-                { "eu_ES_PREEURO", "eu_ES_PREEURO", "eu_ES@currency=ESP" },
-                { "fi_FI_PREEURO", "fi_FI_PREEURO", "fi_FI@currency=FIM" },
-                { "fr_BE_PREEURO", "fr_BE_PREEURO", "fr_BE@currency=BEF" },
-                { "fr_FR_PREEURO", "fr_FR_PREEURO", "fr_FR@currency=FRF" },
-                { "fr_LU_PREEURO", "fr_LU_PREEURO", "fr_LU@currency=LUF" },
-                { "ga_IE_PREEURO", "ga_IE_PREEURO", "ga_IE@currency=IEP" },
-                { "gl_ES_PREEURO", "gl_ES_PREEURO", "gl_ES@currency=ESP" },
-                { "it_IT_PREEURO", "it_IT_PREEURO", "it_IT@currency=ITL" },
-                { "nl_BE_PREEURO", "nl_BE_PREEURO", "nl_BE@currency=BEF" },
-                { "nl_NL_PREEURO", "nl_NL_PREEURO", "nl_NL@currency=NLG" },
-                { "pt_PT_PREEURO", "pt_PT_PREEURO", "pt_PT@currency=PTE" },
-                { "de__PHONEBOOK", "de__PHONEBOOK", "de@collation=phonebook" },
```

Comment:
instead of delete the following tests, could you instead put in the new expectation values instead?

Reply:
done